lib/Common/ConfigFlagsList.h

```diff
@@ -1092,93 +1093,93 @@ FLAGNR(Boolean, Force32BitByteCode, "Force CC to generate 32bit bytecode intende
 
 FLAGNR(Boolean, CollectGarbage        , "Enable CollectGarbage API", DEFAULT_CONFIG_CollectGarbage)
 
-FLAGR (Boolean, Intl                  , "Intl object support", DEFAULT_CONFIG_Intl)
-FLAGNR(Boolean, IntlBuiltIns          , "Intl built-in function support", DEFAULT_CONFIG_IntlBuiltIns)
-FLAGNR(Boolean, IntlPlatform          , "Make the internal Intl native helper object visible to user code", DEFAULT_CONFIG_IntlPlatform)
+FLAGR(Boolean, Intl, "Intl object support", DEFAULT_CONFIG_Intl)
```

Comment:
Please can you revert these whitespace changes; the large gaps are intentional; the idea is to show the descriptions of related fields in line with each other - it's not been done perfectly but would rather keep it.